docs/src/content/docs/utilities/has-integration.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: hasIntegration
+description: Checks whether an integration is already installed
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+`hasIntegration` checks whether an integration has already been installed and simply returns a boolean.
```

Comment:
```suggestion
`hasIntegration` checks whether an integration is already been part of the Astro config. For example:
```

Reply:
What about "`hasIntegration` checks whether an integration is already been installed in the Astro config. For example:" ?

Reply:
mmh not sure about the official terminology here, wait a sec

Reply:
Let's go for `add`

Reply:
yeah sounds good